src/index.html

```diff
@@ -5,8 +5,351 @@
+          <img
+            class="product-card__imac-image"
+            src="images/imac.jpeg"
+            alt="imac-photo"
+          >
+          <h2 class="product-card__product-description" >APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="product-card__product-code">Product code: 195434</p>
+
+          <div class="product-card__feadback">
+            <div class="stars stars--4">
+              <a href="#" class="stars__star"></a>
+              <a href="#" class="stars__star"></a>
+              <a href="#" class="stars__star"></a>
+              <a href="#" class="stars__star"></a>
+              <a href="#" class="stars__star stars__star--last-star"></a>
+            </div>
+
+            <p class="product-card__reviews-text" >Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-text">Price:</p>
+
+            <p class="product-card__price-amount">$2,199</p>
+          </div>
+
+          <a
+            data-qa="card-hover"
+            href="#"
+            class="product-card__button-buy"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div data-qa="card" class="product-card">
+          <img
+            class="product-card__imac-image"
+            src="images/imac.jpeg"
+            alt="imac-photo"
+          >
+          <h2 class="product-card__product-description" >APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="product-card__product-code">Product code: 195434</p>
+
+          <div class="product-card__feadback">
+            <div class="stars stars--4">
+              <a href="#" class="stars__star"></a>
+              <a href="#" class="stars__star"></a>
+              <a href="#" class="stars__star"></a>
+              <a href="#" class="stars__star"></a>
+              <a href="#" class="stars__star stars__star--last-star"></a>
+            </div>
+
+            <p class="product-card__reviews-text" >Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-text">Price:</p>
+
+            <p class="product-card__price-amount">$2,199</p>
+          </div>
+
+          <a
+            data-qa="card-hover"
+            href="#"
+            class="product-card__button-buy"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div data-qa="card" class="product-card">
+          <img
+            class="product-card__imac-image"
+            src="images/imac.jpeg"
+            alt="imac-photo"
+          >
+          <h2 class="product-card__product-description" >APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="product-card__product-code">Product code: 195434</p>
+
+          <div class="product-card__feadback">
+            <div class="stars stars--4">
+              <a href="#" class="stars__star"></a>
```

Comment:
Stars is not a link, use the div instead
```suggestion
              <div class="stars__star"></div>
```